src/main/scala/com/airbnb/scheduler/config/SchedulerConfiguration.scala

```diff
@@ -140,4 +140,8 @@ trait SchedulerConfiguration extends ScallopConf {
   lazy val reconciliationInterval = opt[Int]("reconciliation_interval",
     descr = "Reconciliation interval in seconds",
     default = Some(600))
+
+  lazy val useCassandraLogJobStats = opt[Boolean]("use_cassandra_log_job_stats",
```

Comment:
Rather than having a boolean, we should accept a list of reporters (perhaps by class name?) which implement the same interface, and allow the stats to be written to all or none of them simultaneously.